docs/user/resources/04-30-30-gcp-nfs-backup-schedule.md

```diff
@@ -0,0 +1,71 @@
+# GcpNfsBackupSchedule Custom Resource
+
+The `gcpnfsbackupschedule.cloud-resources.kyma-project.io` custom resource (CR) represents the user-defined schedule for creating backup
+of the `GcpNfsVolume` instances at regular intervals. 
+- Creates the backups by creating the `gcpnfsvolumebackup.cloud-resources.kyma-project.io` resources on the specified interval.
+- Allows users to specify days and times in the form of CRON expressions to automatically create the backups. 
+- Automatically deletes the backups when the backup reaches the configured `maximum retention days`. 
```

Comment:
```suggestion
- Automatically deletes the backups when the backup reaches the configured maximum retention days value. 
```

Reply:
If "maximum retention days" is a field/parameter, use bold font for it. If not, no special formatting is necessary. For more formatting guidelines, please see [03-formatting.md](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md).